app/api/nft/helpers.ts

```diff
@@ -1,14 +1,33 @@
 import { request } from 'graphql-request'
 import { TreeStage, treeStages } from './data'
-import { OwnershipsQuery, UpdateMetadataQuery } from './queries'
-import { Ownership, Nft } from '@kairosnfts/dapp'
+import { GetNftQuery, OwnershipsQuery, UpdateMetadataQuery } from './queries'
+import { Query, Nft } from '@kairosnfts/dapp'
 
 export const auth = `Basic ${Buffer.from(
   process.env.KAIROS_API_KEY! // Keep this secret from the client!
 ).toString('base64')}`
 
+export const getNft = async (nftId: string) => {
+  const data: Query = await request(
+    process.env.NEXT_PUBLIC_KAIROS_API_URL!,
+    GetNftQuery,
+    {
+      nftId: nftId,
+    },
+    {
+      Authorization: auth,
+    }
+  )
+
+  if (!data) {
+    throw new Error('No NFT found')
+  }
+
+  return data.nft
+}
+
 export const getNftsOfUser = async (sessionToken: string) => {
-  const data: { collectorOwnershipsByCollection: Ownership[] } = await request(
+  const data: Query = await request(
```

Comment:
? Removed typing of response why?

Reply:
I dunno, I guess I thought `Query` is more appropriate and cleaner to use here for devs not familiar with all the Query returns. You get all the correct typings still, as we are returning `data.collectorOwnershipsByCollection`. 

I see it like this... devs import `Query` and `Mutation` types, and then when they hover over the type, they see the possible returns (since typescript doesn't know which query we're running anywhere) rather than digging through the exported types in the library to construct the right response type themselves. 

<img width="635" alt="Screenshot 2023-04-28 at 11 44 04 AM" src="https://user-images.githubusercontent.com/6920066/235205835-877e968d-3afe-4e86-a4ff-93235c580af0.png">

But I won't die on this hill 😆 if you guys would rather it be set as `{ collectorOwnershipsByCollection: Ownership[] }`

Reply:
@pistachiomatt Forgot to tag you in the reply.